packs/spells/knock.json

```
@@ -21,7 +21,7 @@
             "value": {}
         },
         "description": {
-            "value": "<p>You make the target easier to open. Knock grants a +4 status bonus to any creature that tries to open the target door, lock, or container with an Athletics or a Thievery check. You can attempt a Thievery check to open the target as part of casting knock, and you add your level even if you're untrained.</p>\n<p>Knock counteracts <em>@UUID[Compendium.pf2e.spells-srd.Item.Lock]</em>.</p>"
+            "value": "<p>You make the target easier to open. <em>Knock</em> grants a +4 status bonus to any creature that tries to open the target door, lock, or container with an @Check[type:athletics|traits:action:open] or a @Check[type:thievery|traits:action:open] check. You can attempt a Thievery check to open the target as part of casting <em>knock</em>, and you add your level even if you're untrained.</p>\n<p><em>Knock</em> counteracts <em>@UUID[Compendium.pf2e.spells-srd.Item.Lock]</em>.</p>\n<p>@UUID[Compendium.pf2e.spell-effects.Item.Spell Effect: Knock]</p>"
```

```suggestion
            "value": "<p>You make the target easier to open. <em>Knock</em> grants a +4 status bonus to any creature that tries to open the target door, lock, or container with an Athletics or a Thievery check. You can attempt a Thievery check to open the target as part of casting <em>knock</em>, and you add your level even if you're untrained.</p>\n<p><em>Knock</em> counteracts <em>@UUID[Compendium.pf2e.spells-srd.Item.Lock]</em>.</p>\n<p>@UUID[Compendium.pf2e.spell-effects.Item.Spell Effect: Knock]</p>"
```
Should be the force open action for athletics or pick a lock with thievery, no need to link them in the spell here since the GM can send to chat from the hazard / journal.

---

Good call!